vault/logical_system.go

```diff
@@ -1714,16 +1714,26 @@ func (b *SystemBackend) handleEnableAuth(
 // handleDisableAuth is used to disable a credential backend
 func (b *SystemBackend) handleDisableAuth(
 	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
-	suffix := strings.TrimPrefix(req.Path, "auth/")
-	if len(suffix) == 0 {
-		return logical.ErrorResponse("path cannot be blank"), logical.ErrInvalidRequest
+	path := data.Get("path").(string)
+	path = sanitizeMountPath(path)
+	fullPath := credentialRoutePrefix + path
```

Comment:
The path that comes in will be something like `auth/token`. Before, we would strip the `auth/` part, then sanitize, so what goes into `disableCredential` would be `token/`. If I am reading this correctly, because it no longer trims the prefix, path will be `auth/token/` and fullPath will be `auth/auth/token/`.